node/Makefile

```diff
@@ -153,7 +153,8 @@ DATE:=$(shell date -u +'%FT%T%z')
 LDFLAGS= -X $(PACKAGE_NAME)/pkg/lifecycle/startup.VERSION=$(GIT_VERSION) \
 	-X $(PACKAGE_NAME)/buildinfo.GitVersion=$(GIT_DESCRIPTION) \
 	-X $(PACKAGE_NAME)/buildinfo.BuildDate=$(DATE) \
-	-X $(PACKAGE_NAME)/buildinfo.GitRevision=$(GIT_COMMIT)
+	-X $(PACKAGE_NAME)/buildinfo.GitRevision=$(GIT_COMMIT) \
+	$(RELEASE_LDFLAGS)"
```

Comment:
Please remove the quote from this line too. To make our build targets consistent, we want to remove quotes in any of the LDFLAGS variable declarations, and then place the quotes around the LDFLAGS variable inside of the build command.

Reply:
Done 👍